src/pages/Pool.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-  <div class="container mx-auto px-4 lg:px-0 pt-8">
+  <div class="pool-container lg:px-4 mx-auto pt-8">
```

Comment:
Again, I think we should stick to the default `container` class. Can you explain the reasoning behind this change?

Reply:
As above, I'd prefer fluid layouts with one final max-width than one with arbitrary max-widths across different breakpoints.